src/Target.h

```diff
@@ -82,6 +82,7 @@ struct Target {
         CLHalf = halide_target_feature_cl_half,
         OpenGL = halide_target_feature_opengl,
         OpenGLCompute = halide_target_feature_openglcompute,
+        EGL = halide_target_feature_egl,
```

Comment:
This needs to be added to the relevant table in PyEnums.cpp as well

Reply:
done